requirements.txt

```diff
@@ -1,18 +1,25 @@
 boto==2.48.0
+certifi==2018.4.16
 cssselect==1.0.3
 cssutils==1.0.2
 Django==1.10
 django-appconf==1.0.1
 django-compressor==2.1
+django-nose==1.4.5
 django-ses-backend==0.1.1
 djmail==0.11.0
 futures==3.0.3
 lxml==4.2.1
 mock==2.0.0
+nose==1.3.7
 pbr==4.0.2
+pipenv==11.10.0
 premailer==2.9.6
 psycopg2==2.7.4
+pytz==2018.4
 rcssmin==1.0.6
 requests==2.9.1
 rjsmin==1.0.12
 six==1.11.0
+virtualenv==15.2.0
```

Comment:
Do we need these last two dependencies for the build to pass?
We should try to keep changes to the requirement file at the minimum.